lib/cn/other_iterator.c

```diff
@@ -0,0 +1,82 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright (C) 2022 Micron Technology, Inc.  All rights reserved.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include <hse_util/element_source.h>
+#include <hse_util/event_counter.h>
+#include <hse_util/hse_err.h>
+
+#include "kvset.h"
+#include "kvset_internal.h"
+#include "omf.h"
+
+struct other_iterator {
+    struct kvset *oi_ks;
+    bool oi_eof;
+    uint32_t oi_offset;
+    struct element_source oi_es;
```

Comment:
rearrange to get better packing?

Reply:
What did you have in mind? I am seeing 56 bytes is the lowest I can go.

Reply:
general rule:  largest to smallest.  Don't know offhand what element_source is, but i would arrange as:
```
    struct kvset *oi_ks;
    struct element_source oi_es;
    uint32_t oi_offset;
    bool oi_eof;
```